tests/utils.go

```diff
@@ -4590,7 +4590,7 @@ func WaitForConfigToBePropagatedToComponent(podLabel string, resourceVersion str
 			return compareResourceVersions(resourceVersion, result["config-resource-version"].(string))
 		}
 		return true
-	}, 10*time.Second, 1*time.Second).Should(BeTrue(), "Not all kubevirt components picked up the kubevirt config successfully")
+	}, 10*time.Second, 1*time.Second).Should(BeTrue(), "Not all kubevirt components picked up the kubevirt config successfully for pod label "+podLabel)
```

Comment:
I think we could change the signature of the function to return an error. This error could include more context(for example Pod`s name).

WDYT?

Reply:
Sounds good!

Reply:
@xpivarc @iholder-redhat I think this is a good idea, but it think it deserves a separate PR of refactoring context. Let's take this change so we will be able to find the root cause of the CI issues sooner.

Reply:
@xpivarc @iholder-redhat I see that code was already updated. Please ignore my previous comment. I just didn't see until now that the `Eventually` can return an error and treated by `ShouldNot(HaveOccured())